webapp/pages/post/_id/_slug/index.vue

```diff
@@ -197,7 +197,7 @@ export default {
 
     .ds-card-image {
       img {
-        max-height: 300px;
+        height: 300px;
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-26T06:45:49Z" title="Thursday, September 26th 2019, 8:45:49 am +02:00">Sep 26, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -193,7 +193,7 @@ export default {
 
     .ds-card-image {
       img {
-        max-height: 300px;
+        height: 300px;
```


code creep :fearful: 
I would prefer not having unrelated changes in `PR`s... we actually have talked, and there might even be an issue to remove the `max-height` altogether... enforcing a height of `300px` would be going in the opposite direction..... please revert this change, and let's make it in a separate ``PR`, please

Reply:
> [<img alt="vbelolapotkov" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/vbelolapotkov) **Authored by [vbelolapotkov](https://github.com/vbelolapotkov)**
_<time datetime="2019-09-26T07:19:07Z" title="Thursday, September 26th 2019, 9:19:07 am +02:00">Sep 26, 2019</time>_

---


@mattwr18 I wouldn't say this is a code creep. My bad of not providing the note on the intent of this change. So here it is. 

When you navigate to the `<post_url>#comments` from `<post_url>/more-info` there is a race condition takes place between loading post image and scrolling comments into view.
If the image is loaded after scrolling is finished it will push comments section 300px down which looks weird especially in mobile.

So fixing the image height solves this issue. 
I understand that this can affect the posts with images less than 300px in height but I doubt anybody uploads such a small post hero images. FYI, I've checked that posts without images are not affected.

Probably, a better solution would be using a placeholder image while the original one is being loaded.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-27T07:04:12Z" title="Friday, September 27th 2019, 9:04:12 am +02:00">Sep 27, 2019</time>_

---


Thank you for your explanation @vbelolapotkov... what would happen if we then implemented the no max-height of an image, which has been requested? the scrolling functionality would break?
Or we'd have to follow your suggestion of adding a placeholder image with a fixed height of 300px?